### ISSUE TITLE
Confusion on embedding.opengl adaptation

### ISSUE THREAD

**Issue by jgoth** [neo4j-graph-analytics/ml-models#9]
There exists a 1.0.2 tag, but the project seems to only build a 1.0.1 version of the jar file. Also, whether I download the existing 1.0.1 release or build the 1.0.2 tagged version of the code, then upload the jar file into the plugins folder, and update the neo4j.conf file as such:

dbms.security.procedures.whitelist=regression.*,embedding.*
dbms.security.procedures.unrestricted=regression.*,embedding.*

I'm getting a ProcedureNotFound error. It seems as if others are able to utilize the embedding.opengl procedures with some degree of success. I'm genuinely curious as to what I'm doing wrong.

**Comment by djsauble** [comment]
I'm seeing the same problem. Here's the [debug.log output](https://gist.github.com/djsauble/5d3b79367effa77dc294507d2ffc9ac4#file-debug-log-L366) from my Neo4j 3.5.0 instance.

In particular, Neo4j seems to have trouble loading embedding.DeepGL from the jar, along with a bunch of other jars/classes. Interestingly, the regression.* procedures do load successfully (according to `CALL dbms.procedures()`).

I've also tried installing this plugin on Neo4j 3.4.7 and 3.3.7 with the same results.

**Comment by Mursy** [comment]
@djsauble  I have the exact problem. I can see the regression.* procedures but not the embedding.* ones. Did you find a solution for this?

**Comment by djsauble** [comment]
@Mursy Nope, I gave up. Their test suite isn't runnable either (has dependencies on test data that isn't part of the repo), so it's difficult to tell where the problem is.

**Comment by mneedham** [comment]
Hey,

You need to install the graph algorithms library to have the embedding procedure show up. I updated the readme to explain how to do that.

Cheers, Mark

**Comment by Mursy** [comment]
@djsauble @mneedham Thank you for the replies. I will check it again now and let you know how it goes.

Can you please provide the link for the updated readme? Can't seem to find the updated one.

Thank you again.
